Create docs for data migrations (#2593)
port-labs/port-docs: docs/build-your-software-catalog/customize-integrations/configure-data-model/migrating-data/migrating-data.md

---
sidebar_position: 3

> title: Migrating data
> sidebar_label: Migrating data
> ---
> 
> import ApiRef from "/docs/api-reference/\_learn_more_reference.mdx"
> 
> import Tabs from "@theme/Tabs"
> import TabItem from "@theme/TabItem"
> 
> # Migrating Blueprint Data

Reviewer: ```suggestion
title: Migrate data
sidebar_label: Migrate data
---

import ApiRef from "/docs/api-reference/\_learn_more_reference.mdx"

import Tabs from "@theme/Tabs"
import TabItem from "@theme/TabItem"

# Migrate Blueprint Data
```

When evolving your software catalog, you may need to migrate existing [entities](/build-your-software-catalog/sync-data-to-catalog/#entities) to reflect changes in your [blueprint](/build-your-software-catalog/customize-integrations/configure-data-model/setup-blueprint/) structure. Port provides a simple approach to migrate all entities of a blueprint efficiently and safely.

## When to migrate data

Data migration becomes necessary when you need to:

- **Transform data format**: Change property values and/or merge and seperate properties in existing blueprints.
- **Update relations**: Modify existing relations between blueprints.

:::info Blueprint schema evolution
After migrating data, if you want to revert the changes, you will need to create a new migration from scratch.
:::

## How to migrate data

### Using the UI

> To migrate data using Port UI, go to the **Builder Page** and select your desired **Blueprint**.  

Reviewer: ```suggestion
To migrate data using Port UI, go to the [Builder page](https://app.getport.io/settings/data-model) of your portal and select your desired **blueprint**.  
```

> Click the three dots menu of the blueprint you want to migrate, and select **Migrate data** from the dropdown menu.

Reviewer: ```suggestion
Click on the `...` button of the blueprint you want to migrate, and select `Migrate data` from the dropdown menu.
```

> ### Visual walkthrough: Migrating data in Port

Reviewer: since this header is on the same level as the "Using the UI" one, it looks like another option. I think we can just remove this header since the content refers to the previous one.

Below are some example screenshots to help you understand the migration process in Port UI.
#### Selecting the source blueprint

> ![Selecting the source blueprint](/img/build-your-software-catalog/custom-integrations/configue-data-model/migrating-data/three-dots-on-blueprint-in-builder.png)

Reviewer: please make the images smaller. You can use this format:
`<img src='/img/self-service-actions/advancedInputsFormExample.png' width='60%' />`
